README.md

```diff
@@ -1,6 +1,9 @@
 blackbird
 =========
 
+[![Circle CI](https://circleci.com/gh/Vagrants/blackbird/tree/master.svg?style=shield)](https://circleci.com/gh/Vagrants/blackbird/tree/master)
+[![License](https://img.shields.io/badge/license-WTFPL-blue.svg)](http://www.wtfpl.net/about/)
+
```

Comment:
Wow! amazing icons!
OK, remove license record at end of README and use below icon style(such as circle build icon).
coloring.... Hmm... I'd like to blue,too!
Thanks for good information about license record.